backends/arm/test/common.py

```diff
@@ -122,10 +125,21 @@ def get_u55_compile_spec(
     ).build()
 
 
+def get_u85_compile_spec(
```

Comment:
Any reason we are not using this in any tests - at least till TOSA? Or it's in the future PRs?

Reply:
Yes, and there is a separate PR being prepared to enable unit test for this compile spec as well.